cmake/SetupThirdParty.cmake

```diff
@@ -63,10 +63,16 @@ if (TARGET mfem)
     # Note - white238: I can't seem to get this to pass install testing due to mfem being included
     # in multiple export sets
     message(STATUS "MFEM support is ON, using existing mfem target")
+
     # Add it to this export set but don't prefix it with tribol::
-    install(TARGETS              mfem
-            EXPORT               tribol-targets
-            DESTINATION          lib)
+    # NOTE: imported targets cannot be part of an export set
+    get_target_property(_is_imported mfem IMPORTED)
```

Comment:
should this pattern be used for axom as well?

Reply:
Already done.. though this shouldn't happen again with several improvements. I don't even think it happens on Axom develop atm.

https://github.com/LLNL/axom/pull/1270

Reply:
Actually, I meant in the import of axom in this file (above).... in the case that someone is trying to import axom targets?

Reply:
When I wrote this fix, I was unable to reproduce this in Axom's develop testing against our install example. `mfem` has been namespaced as `axom::mfem` for a while now.